### PR TITLE
work on #14: initial dashboard implemetation

### PR DESCRIPTION
**Pull request by monicachew**
Getting this into a PR so it's easier to find.

### CODE REVIEW

**Comment by monicachew** [comment]
I worked on this a little bit then decided that it's better to preprocess the data from the DB -- since the json is 5M it doesn't make sense for the client to do all the work. But that's not very important perhaps.

**Comment by monicachew** [comment]
I think a reasonable workflow would be a cronjob to update the CT DB every day or so, run the go script, then dump the DB in the dashboard directory.

**Comment by monicachew** [comment]
This is now up at http://people.mozilla.org/~mchew/dashboard/. I omitted the validPeriodTooLong because the casing was incorrect in the go tool, but I can put that back after I re-run it.

Things to do:
1) Probably dump to different files, the output of db2json is now back up to 6.9M and we haven't dumped the series for "worst" offenders yet.
2) Figure out if/how we want to switch between raw/normalized scores.
3) Figure out the timestamp truncation issue, supposedly issuerReputation should only have beginTimes at the start of the month.

File: dashboard/index.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Issuer Reputation Dashboard</title>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.8.2/jquery.min.js"></script>
+  <script src="https://code.highcharts.com/stock/highstock.js"></script>
+  <script src=series.js></script>
```
Comment (mozkeeler) [inline dashboard/index.html]: nit: let's do 'src="series.js"'

**Comment by mozkeeler** [comment]
We probably don't need  dashboard/issuerReputation.json now, right?

**Comment by monicachew** [comment]
Probably not. That whole change needs cleanup, there are some bad names.
Also we may need a sidebar(?) or something similar to toggle between stuff.

File: tools/sunlight.go [hunk 2]
```diff
@@ -125,7 +125,7 @@ func main() {
 	 insert into issuerReputation(
 		issuer,
 		issuerInMozillaDB,
-		validPeriodTooLongNormalizedScore, validPeriodTooLongRawscore,
+		validPeriodTooLongNormalizedScore, validPeriodTooLongRawScore,
```
Comment (mozkeeler) [inline tools/sunlight.go]: Does this also need to be fixed in the create table statement? (line 75, but github won't let me comment there)

**Comment by mozkeeler** [comment]
Right - I was thinking the sidebar could be a next step once we've landed this initial work. Do you want to do the cleanup or should I go ahead?

**Comment by monicachew** [comment]
Please go ahead, I'm not sure I have time to work on this until next week.

**Comment by mozkeeler** [comment]
This works for me. I think the dates issue was fixed in c1f467f5 - try rebuilding sunlight.go & tools/sunlight.go and re-generating the DB?
In the meantime, I'm going to merge this so there's something to work off of in terms of smaller improvements.
